hack/table-gen/Makefile

```diff
@@ -0,0 +1,11 @@
+tracepipeline:
+	go run table-gen.go --crd-filename ../../installation/resources/crds/telemetry/tracepipelines.crd.yaml --md-filename ../../docs/05-technical-reference/00-custom-resources/telemetry-03-tracepipeline.md --crd-title TracePipeline
+
+logpipeline:
+	go run table-gen.go --crd-filename ../../installation/resources/crds/telemetry/logpipelines.crd.yaml --md-filename ../../docs/05-technical-reference/00-custom-resources/telemetry-01-logpipeline.md --crd-title LogPipeline
+
+logparser:
+	go run table-gen.go --crd-filename ../../installation/resources/crds/telemetry/logparsers.crd.yaml --md-filename ../../docs/05-technical-reference/00-custom-resources/telemetry-02-logparser.md --crd-title LogParser
+
+.PHONY: run
+run: tracepipeline logparser logpipeline
```

Comment:
What if other teams want to use it as well? Would they add the mapping to the run target as well, or should we have a target per domain? Maybe let's call it something like "make telemetry-docs"? and prefix the individual targets with telemetry as well? @skhalash what do you think?

Reply:
change to 'generate'